util/capacity-reservation.py

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env python3
```

Comment:
Wondering why you decided to create your own client instead of just using the awscli? https://docs.aws.amazon.com/cli/latest/reference/ec2/create-capacity-reservation.html

Reply:
Good question. I don't have a very strong answer. I got started just writing boto3 scripts becuase the image doesn't have awscli on it. I can add awscli in but I will need to test there's no conflicts. Another reason is standardizing certain actions. However I supposed if I had awscli installed on the jumpbox I could just put these in a README.

Reply:
I don't have a custom built AMI yet. I will file an issue on this repo to see if I can add awscli to the template with no problems -- iirc when I added it before there were unexpected issues.